napari/layers/utils/plane.py

```diff
@@ -0,0 +1,37 @@
+from typing import Tuple
+
+import numpy as np
+from pydantic import validator
+
+from ...utils.events import EventedModel
+from ...utils.misc import ensure_n_tuple
+
+
+class Plane3D(EventedModel):
+    """Object modelling a plane in 3D with a defined thickness.
+
+    Attributes
+    ----------
+    position : 3-tuple
+        A position in 3D, defined in data coordinates.
+    normal_vector : 3-tuple
+        A normal vector in 3D, defined in data coordinates.
+    thickness : float
```

Comment:
If we want to be able to rotate the plane as the viewer dims roll/transpose, we might want to add an attribute that gives the axis each dimension in the plane corresponds with. For example a list where each element is the layer data axis that index in the position/normal corresponds with.